lib/cache.js

```diff
@@ -51,14 +50,9 @@ Cache.prototype.clr = function(
   });
 };
 
-const cache = (
-  // Extend Map interface with:
-  // `cache.allocated` - total allocated size
-  // `cache.add(key, val)` - add record
-  // `cache.del(key)` - delete record
-  // `cache.clr(prefix, fn)` - delete all if `key.startsWith(prefix)`
-  // Returns: object, cache instance
-) => (new Cache());
+// Create Cache, enhanced Map
+// Returns: <object>, Cache instance
```

Comment:
Same here, I think `Object` will be better:
```suggestion
// Returns: <Object>, Cache instance
```